.ci/macos/dependencies/chrome.sh

```
@@ -0,0 +1,2 @@
+# Check chrome version
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)